### PR TITLE
Format markdown

### PR DESCRIPTION
**Pull request by mattmoor-sockpuppet**
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github)`
/assign @samodell

### CODE REVIEW

File: community/samples/serving/helloworld-r/README.md [hunk 1]
```diff
@@ -58,37 +58,38 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-r
            return "Error in R script.", 500
 
        return output
+   ```
 
+if **name** == "**main**":
+app.run(debug=True,host='0.0.0.0',port=int(os.environ.get('PORT', 8080)))
```
Comment (mattmoor) [inline community/samples/serving/helloworld-r/README.md]: This creates a bad diff that we must manually fix.

File: docs/install/Knative-custom-install.md [hunk 2]
```diff
@@ -71,24 +71,40 @@ with Knative.
...
 - **Serving Component and Observability Plugins**:
-  - https://github.com/knative/serving/releases/download/{{< version >}}/serving.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-logs-elasticsearch.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-metrics-prometheus.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-jaeger.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-jaeger-in-mem.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-zipkin.yaml
-  - https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-tracing-zipkin-in-mem.yaml
+  - https://github.com/knative/serving/releases/download/{{<
```
Comment (RichieEscarez) [inline docs/install/Knative-custom-install.md]: These are all broken too - the shortcode must remain on a single line 
`{{< version >}}`

Comment (mattmoor) [reply docs/install/Knative-custom-install.md]: Is there any way for it to not have spaces?

Comment (RichieEscarez) [reply docs/install/Knative-custom-install.md]: no not for shortcodes in Hugo (i tried w/o spaces but they result in printing directly to the ouput - hugo wont process it)

Comment (RichieEscarez) [reply docs/install/Knative-custom-install.md]: Related FYI: https://github.com/knative/docs/issues/1678

**Pull request closed without merging**